docs/paper/evidence/web-2026-04-06/gemma_3_model_card.html

```
@@ -0,0 +1,75 @@
+          href="https://ai.google.dev/gemma/docs/core/model_card_3?hl=ru" /><link rel="alternate" hreflang="es-419"
+          href="https://ai.google.dev/gemma/docs/core/model_card_3?hl=es-419" /><link rel="alternate" hreflang="th"
+          href="https://ai.google.dev/gemma/docs/core/model_card_3?hl=th" /><link rel="alternate" hreflang="tr"
+          href="https://ai.google.dev/gemma/docs/core/model_card_3?hl=tr" /><link rel="alternate" hreflang="vi"
+          href="https://ai.google.dev/gemma/docs/core/model_card_3?hl=vi" /><link rel="alternate" hreflang="sq"
+          href="https://ai.google.dev/gemma/docs/core/model_card_3?hl=sq" /><title>Gemma 3 model card &nbsp;|&nbsp; Google AI for Developers</title>
+
+<meta property="og:title" content="Gemma 3 model card &nbsp;|&nbsp; Google AI for Developers"><meta property="og:url" content="https://ai.google.dev/gemma/docs/core/model_card_3"><meta property="og:image" content="https://ai.google.dev/static/site-assets/images/share.png">
+  <meta property="og:image:width" content="1200">
+  <meta property="og:image:height" content="675"><meta property="og:locale" content="en"><meta name="twitter:card" content="summary_large_image"><!-- stripped telemetry script: Google Tag Manager / Analytics -->
+      
+    </devsite-analytics>
```

HTMLHint (1.9.2)

[error] Tag must be paired, no start tag: [ </devsite-analytics> ]

(tag-pair)